kong/plugins/cors/handler.lua

```diff
@@ -114,15 +114,14 @@ function CorsHandler:access(conf)
       configure_methods(ngx, conf)
       configure_max_age(ngx, conf)
 
-      -- Don't add response headers because we already added them all
-      ngx.ctx.skip_response_headers = true
-
       return responses.send_HTTP_NO_CONTENT()
     end
 
     -- Don't add any response header because we are delegating the preflight to
     -- the upstream API (conf.preflight_continue=true)
     ngx.ctx.skip_response_headers = true
+  else
+    ngx.ctx.skip_response_headers = false
```

Comment:
Regardless of the behavioral change introduced in this PR as pointed by @p0pr0ck5, I don't think explicitly setting this to false is making this code any more readable than it was before. If anything it is adding a branch and an assignment that aren't needed after all.